scripts/apps/web-publisher/controllers/WebPublisherManagerController.js

```diff
@@ -44,6 +45,9 @@ export function WebPublisherManagerController($scope, publisher, modal) {
         $scope.newSite = {};
         self.openSiteModal = !self.openSiteModal;
         publisher.setTenant('default');
+        if (!self.openSiteModal) {
+            $rootScope.$broadcast('refresh:routes');
```

Comment:
imo this should not be `$rootScope`. I would limit it to the $scope of the `SiteRoutesDirective` or find another way to communicate this change. `$rootScope` events concern the whole application, while this only concerns a mere directive.